contrib/Open3DGC/o3dgcSC3DMCDecoder.inl

```diff
@@ -72,9 +72,12 @@ namespace o3dgc
         unsigned char mask = bstream.ReadUChar(m_iterator, m_streamType);
 
         ifs.SetCCW             ((mask & 1) == 1);
-        ifs.SetSolid           ((mask & 2) == 1);
-        ifs.SetConvex          ((mask & 4) == 1);
-        ifs.SetIsTriangularMesh((mask & 8) == 1);
+        // (mask & 2) == 1
+        ifs.SetSolid           (false);
+        // (mask & 4) == 1
+        ifs.SetConvex          (false);
+        // (mask & 8) == 1
+        ifs.SetIsTriangularMesh(false);
```

Comment:
I tried to find the implementation inside AMD's official repository and they seems to have changed.
https://github.com/amd/rest3d/blob/ecc287150791c1d5aef10f0606de46b2147c0944/server/o3dgc/src/o3dgc_decode_lib/src/o3dgcDynamicVectorDecoder.cpp#L118-L150

Either this part does not exist or this is changed to suit assimp.

Reply:
I guess we need to update it.